tasks/connectors/snyk_v2/snyk_v2_task.rb

```diff
@@ -125,9 +124,11 @@ def run(opts)
               }"
               print_debug "issue filter json = #{issue_filter_json}"
 
-              issue_json << client.snyk_get_issues(@page_size, issue_filter_json, page_num, @from_date, @to_date)
-              print_debug "issue json = #{issue_json}"
+              page_num += 1
+              issue_json << client.snyk_get_issues(@page_size, issue_filter_json, page_num, @from_date, @to_date) unless
```

Comment:
The code looks good, but you could reduce the large of this line doing something like this:

`issue_json << client.snyk_get_issues(@page_size, issue_filter_json, page_num, @from_date, @to_date).compact`

this is gonna remove all `nil` values what would come from the method, so at the end of the code block the `array` will be flattened so that´s gonna remove all empty values.

Reply:
Thanks, that's a good idea, but it doesn't work in this case